internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -213,3 +245,11 @@ func setVariable(variables VariableMap, name string, value interface{}) error {
 	variables[name] = apiextensionsv1.JSON{Raw: marshalledValue}
 	return nil
 }
+
+func toJSONCompact(value string) apiextensionsv1.JSON {
+	var compactValue bytes.Buffer
+	if err := json.Compact(&compactValue, []byte(value)); err != nil {
+		panic(err)
+	}
+	return apiextensionsv1.JSON{Raw: compactValue.Bytes()}
+}
```

Comment:
I assume there will be only one of this func (either one or the one from @sbueringer PR)

Reply:
Yeah - just the depending on which is merged first I guess.

Reply:
I've added them in "writing a clusterclass" is there somewhere else they should go?

Reply:
And in the proposal. (you can check my PR)